### PR TITLE
Sql

### PR DESCRIPTION
**Pull request by ishimwezachee**
in this project I have done the following ;
- Used [this template](https://github.com/microverseinc/curriculum-template-databases) to generate your repo.
- Created a database named `vet_clinic`.
- Created a table named `animals` with the following columns:
  - `id`: integer
  - `name`: string
  - `date_of_birth`: date
  - `escape_attempts`: integer
  - `neutered`: boolean
  - `weight_kg`: decimal
- Copied the SQL I used in the previous point into the `schema.sql` file in the generated repository.
- Copied the SQL I used in the previous point into the `data.sql` file in the generated repository.
- Copied the SQL I used in the previous point into the `queries.sql` file in the generated repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,19 +13,17 @@ This repository includes files with plain SQL that can be used to recreate a dat
 - Check [queries.sql](./queries.sql) for examples of queries that can be run on a newly created database. **Important note: this file might include queries that make changes in the database (e.g., remove records). Use them responsibly!**
```
Comment (SajjadAhmad14) [inline README.md:14]: Please add a project title and description to the README. This way it looks more professional.

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,34 @@
...
+FROM animals
+WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+-- List the name of all animals that are neutered and have less than 3 escape attempts.
+SELECT name 
+FROM animals
+WHERE neutered = 'true' AND escape_attempts < 3;
+-- List date of birth of all animals named either "Agumon" or "Pikachu".
+SELECT * 
+FROM date_of_birth
+WHERE name = 'Agumon' or name ='Pikachu';
```
Comment (SajjadAhmad14) [inline queries.sql]: Please add a query here to select the date of birth of all animals named either "Agumon" or "Pikachu", so you need to select `date_of_birth` from the `animals` table here.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Zachee, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🎉🍾✔**
Hello @ishimwezachee 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**